rider-fsharp/src/main/java/com/jetbrains/rider/plugins/fsharp/services/fsi/FsiProcessHandler.kt

```diff
@@ -2,12 +2,9 @@ package com.jetbrains.rider.plugins.fsharp.services.fsi
 
 import com.intellij.execution.process.OSProcessHandler
 import com.intellij.openapi.util.Key
-import java.nio.charset.Charset
 
 class FsiProcessHandler (
-        process: Process, commandLine: String?, charset: Charset?)
-    : OSProcessHandler(process, commandLine, charset) {
-
+        process: Process, commandLine: String?) : OSProcessHandler(process, commandLine) {
```

Comment:
The charset is no longer passed. Was it the default one that would be choosed if nothing is passed explicitly?

Reply:
According to https://github.com/JetBrains/intellij-community/blob/b67267ef4ac7181b58e5b0abcc4e43fb886374f9/platform/core-api/src/com/intellij/openapi/vfs/encoding/EncodingRegistry.java#L19-L23 
the default charset configured in settings.

Reply:
It does not answer the question about default value inside `OSProcessHandler`constructor.

Reply:
The encoding could be changed in settings but the default is UTF-8.

Reply:
We set UTF-8 for fsi explicitly here:
https://github.com/JetBrains/fsharp-support/blob/dbee3578ab7ab1170f55481b5169fcfc1b27b325/ReSharper.FSharp/src/FSharp.Psi.Features/src/Fsi/FsiHost.fs#L54
Let's not change the encoding on the console side.